server/app/services/export/PdfExporter.java

```diff
@@ -184,6 +192,26 @@ private byte[] buildPDF(
           document.add(eligibility);
         }
       }
+      if (answersOnlyHidden.size() > 0) {
+        document.add(Chunk.NEWLINE);
+        Paragraph hiddenText =
+            new Paragraph(
+                "Hidden Questions : ", FontFactory.getFont(FontFactory.HELVETICA_BOLD, 12));
```

Comment:
_suggestion_: If there are other section headings in the doc, can we match their font size? It's currently the same font size as a question, which makes it hard to skim.